src/main/resources/init_db.sql

```diff
@@ -0,0 +1,13 @@
+CREATE SCHEMA `storage` DEFAULT CHARACTER SET utf8 ;
+CREATE TABLE `storage`.`manufacturers`
+(
+  id              INT unsigned NOT NULL AUTO_INCREMENT,
+  name            VARCHAR(150) NOT NULL,
+  country         VARCHAR(150) NOT NULL,
+  deleted         bit(1) NOT NULL DEFAULT b'0',
```

Comment:
optional, but why do you choose bit and not tinyint?

Reply:
That's what i found on stackoverflow, and I thought BIT(1) suits better: 

> "A TINYINT is an 8-bit integer value, a BIT field can store between 1 bit, BIT(1), and 64 bits, BIT(64). For a boolean values, BIT(1) is pretty common."